eox_tagging/api/v1/test/test_viewset.py

```diff
@@ -342,6 +344,44 @@ def test_filter_by_type(self, _):
 
         self.assertEqual(response.status_code, 200)
 
+    @patch_permissions
+    def test_filter_using_after_datetime(self, _):
+        """
+        Used to test filtering tags using a range datetime filter. In this case, the filter is
+        before a specific datetime.
+        """
+        query_params = {
+            "activation_date_0": "2020-10-10 10:20:30",
+        }
+
+        response = self.client.get(self.URL, query_params)
+
+        datetime_value = datetime.datetime(2020, 10, 10, 10, 20, 30)
```

Comment:
this line and the next could be in the arrange section.

Reply:
you're right!!